src/db-wrapper.ts

```diff
@@ -0,0 +1,233 @@
+import { Database, aql } from "arangojs";
+import { DocumentCollection } from "arangojs/collection";
+import { Config } from "arangojs/connection";
+
+import { BMT, BMT_IDs } from "./common-types";
+import { RangerConfig } from "./ranger-config";
+
+export interface IDbWrapper {
```

Comment:
I suggest to rename it to the `IBmtDb` because it is specific database. And we dont' need to specify suffixes like `wrapper`, `helper` etc. it is an extra infromational noise.

Reply:
Done